tesseract/tesseract_environment/include/tesseract_environment/core/commands.h

```diff
@@ -129,7 +133,7 @@ class RemoveLinkCommand : public Command
 class RemoveJointCommand : public Command
 {
 public:
-  RemoveJointCommand(const std::string& joint_name) : Command(CommandType::REMOVE_JOINT), joint_name_(joint_name) {}
+  RemoveJointCommand(std::string joint_name) : Command(CommandType::REMOVE_JOINT), joint_name_(std::move(joint_name)) {}
```

Comment:
Is this equivalent to making it const in that the function will not change the contents of the string joint_name?